source/CoordinateConversion/CoordinateConversionLibrary/Models/CoordinateConversionLibraryConfig.cs

```diff
@@ -155,43 +195,21 @@ public void LoadConfiguration()
 
         #region Private methods
 
-        public string GetConfigFolder()
```

Comment:
@saip @dfoll  - this commit seems to have overwritten some changes from a prior PR/commit ( #606 ). Not sure why Git did not flag this as a conflict, but just a word of caution to always update your branch from the upstream (dev) before doing a PR (and maybe even after if the branch/PR has been open for a while and there have been other PRs merged)